apps/passport-server/migrations/71_generic_issuance_semaphore_history.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE generic_issuance_semaphore_history (
+  id BIGSERIAL PRIMARY KEY,
+  pipeline_id UUID NOT NULL,
+  group_id UUID NOT NULL,
+  root_hash VARCHAR NOT NULL,
```

Comment:
should there be a unique constraint on the tuple of these three columns?

Reply:
I'm not totally sure. Can the root hash be the same if someone is added to the group and then removed, so that the group returns to the earlier membership? I think this would set the deleted leaf node to a "zero value", so the hash would be different, but I'm not 100% sure of that.